CONTRIBUTING.md

```diff
@@ -0,0 +1,205 @@
+# Contributing
+
+Thanks for thinking about helping! How would you like to help?
+
+- [I want to build a new feature.](#how-to-tackle-a-new-feature)
+- [I want to improve the documentation.](#what-kind-of-documentation-are-you-writing)
+- [I want to fix a bug.](#how-to-fix-a-bug)
+- [I want to refactor some code.](#how-to-refactor-code)
+- [I don't know how to help.](#what-if-i-dont-know-how-to-help)
+
+## Before you get started
+
+1. [Fork the repo](https://help.github.com/articles/fork-a-repo/).
+
+2. Run the tests. We only take pull requests with passing tests, and it's great to know that you have a clean slate: `bundle && bundle exec rake`
+
+Okay, you're ready to go now. [How do you want to help?](#contributing)
```

Comment:
The link seems unnecessary since it redirects to the small section right above that the user just read. I wound just have: `Okay, now you're ready to go!` and leave it at that.